windows_docker_resources/Dockerfile

```diff
@@ -48,6 +48,9 @@ RUN copy /b C:\TEMP\rticonnextdds-src\rti_connext_dds-6.0.1-pro-target-x64Win64V
 # ROS_DISTRO argument should be set to install dependencies for the target ROS version.
 ARG ROS_DISTRO
 
+# Dinamically set VS_VERSION from CI options
+RUN sed -i 's/"vs_release": .*"/"vs_release": '\"${CI_VISUAL_STUDIO_VERSION}\"'/g' install_ros2_%ROS_DISTRO%.json
```

Comment:
Is there any way we can set this without doing a sed?  Maybe by passing a parameter?  This isn't a blocker, but it would be nicer.

Reply:
Actually that was a brain fart from me, we cannot use sed on windows :laughing: . But the principle stays, the issue with using a parameter is how do we inject that on the chef run.

Reply:
> Actually that was a brain fart from me, we cannot use sed on windows 😆 . But the principle stays, the issue with using a parameter is how do we inject that on the chef run.

That's true, but we must be doing that somehow.  At least, chef seems to know which ros_distro we are using, so there must be a way to pass that information in.